### ISSUE TITLE
API might overload CLN gRPC API

### ISSUE THREAD

**Issue by fusion44** [fusion44/blitz_api#126]
refs https://github.com/rootzoll/raspiblitz/issues/3182

![image](https://user-images.githubusercontent.com/503952/175096504-dcadb47b-5dc8-42d4-9c28-665c9de1647f.png)

This might be the result of a mix of problems:
* /list-all-tx being quite inefficient
* the WebUI polling /list-all-tx every two seconds
* RaspiberryPI being low powered

**Comment by cstenglein** [comment]
@fusion44 should I poll slower?
Like every 10/20s?

**Comment by fusion44** [comment]
LND has an astronomical high CPU usage too. So that's a systemic issue I didn't notice on my dev PC as it is quite a bit faster than a PI.

![image](https://user-images.githubusercontent.com/503952/175644543-5dcee24a-1945-45c4-8448-1d343c2ab064.png)

**Comment by openoms** [comment]
Should find a way to not keep calling functions which return an error.
Agree @cstenglein should slow down significantly. Max  every 20s as we are still building on the RPi as the main platform.
And to be clear this polling only goes on when the webUI is open, right?

**Comment by fusion44** [comment]
Yes, the polling in only going on when a browser tab with the WebUI is open. While I agree that we must get away from polling, I actually believe now that the polling is not the root cause of the problem. 

I've tested a bit more tonight and the problem persists even as the polling stops.

**Comment by fusion44** [comment]
@openoms Have you tried to let the node run for a bit? CPU usage tends to get down to normal levels from time to time.

**Comment by openoms** [comment]
> @openoms Have you tried to let the node run for a bit? CPU usage tends to get down to normal levels from time to time.

this https://github.com/fusion44/blitz_api/issues/125#issue-1277958308 looks like that sent payment record is causing a `JSONDecodeError` and often leading to this line
```
E0625 21:44:06.980009804 1907979 fork_posix.cc:76]           Other threads are currently calling into gRPC, skipping fork() handlers
```
should be rate limited.

**Comment by openoms** [comment]
Testing Open channel with CLN:
```
![image](https://user-images.githubusercontent.com/43343391/176832337-4f41f20f-d9d5-4f66-a612-fe40e79bf78e.png)
```
